examples/pipe/interlock_test.c

```diff
@@ -69,9 +69,10 @@ static void *null_writer(pthread_addr_t pvarg)
   fd = open(FIFO_PATH2, O_WRONLY);
   if (fd < 0)
     {
-      fprintf(stderr, "null_writer: Failed to open FIFO %s for writing, errno=%d\n",
+      fprintf(stderr, "null_writer: "
+              "Failed to open FIFO %s for writing, errno=%d\n",
               FIFO_PATH2, errno);
-      return (void*)1;
+      return (void *)1;
```

Comment:
Do we need FAR here?

Reply:
The whole code base doesn't use FAR, so it's better to move to new patch.